### PR TITLE
Improvement for k8s.io/docs/reference/using-api/api-concepts/

### PR DESCRIPTION
**Pull request by NicoMarksman**

### CODE REVIEW

**Comment by dipankardas011** [comment]
@NicoMarksman Nice catch!

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ HTTP verb for a **patch** is PATCH.
 
 ## Resource URIs
 
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a
-namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource
+All resource types are either scoped by the cluster (`/api/GROUP/VERSION/*`) or to a
+namespace (`/api/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource
```
Comment (liggitt) [inline content/en/docs/reference/using-api/api-concepts.md:82]: `/apis/GROUP/VERSION` is correct for APIs placed in an API group.

Only the original "core" API group uses the `/api/v1/...` form

**Comment by liggitt** [comment]
Thanks for the pull request, but the existing text is correct for all APIs contained in a named API group

/close

**Comment by stephen-dahl** [comment]
@liggitt this MR was intended to solve https://github.com/kubernetes/website/issues/35381
but I think that @NicoMarksman misunderstood the problem. I have added more clarification.
This documentation should explicitly state that core resources use /api instead of /apis.

**Pull request closed without merging**